### PR TITLE
Allow the flipped state to be passed via props

### PR DESCRIPTION
**Pull request by muyiwa-felixson**

### CODE REVIEW

File: FlipCard.js [hunk 1]
```diff
@@ -50,12 +54,12 @@ class FlipCard extends Component{
             outputRange: ['0deg', '360deg', '0deg']
         });
         return (
-            <TouchableOpacity onPress={this._flipToggleCard} style={styles.animatedContainer}>
+            <View onPress={this._flipToggleCard} style={styles.animatedContainer}>
```
Comment (calcazar) [inline FlipCard.js]: This should remain as TouchableOpacity. React-Native doesn't officially support onPress for Views. https://facebook.github.io/react-native/docs/view.html

**Review by calcazar — CHANGES_REQUESTED** [review]
I would just chance the view back to a touchable opacity. If you wanted no opacity I would probably create a prop that controls the opacity level. Maybe even a boolean.

File: FlipCard.js (hunk 1 above)
Comment (muyiwa-felixson) [reply FlipCard.js]: That works too, I would add the option to pass the opacity props to the component.

**Comment by muyiwa-felixson** [comment]
@calcazar kindly review the changes

File: FlipCard.js [hunk 2]
```diff
@@ -47,19 +47,24 @@ class FlipCard extends Component{
             friction: this.props.friction,  // Oscillate a lot
         }).start();
     };
-
+    _getOpacity() {
+        if (!this.props.activeOpacity) {
+            return 1;
```
Comment (calcazar) [inline FlipCard.js]: I would suggest we return the default of 0.2 and the let the user override the default with 1. Everything else looks good~

**Review by calcazar — CHANGES_REQUESTED** [review]
I would make the default to 0.2 and then allow users to override that with the prop

**Comment by muyiwa-felixson** [comment]
@calcazar Note I added prop type validation

**Review by calcazar — APPROVED** [review]

**Pull request closed without merging**